docs/devel/releasing.md

```diff
@@ -93,21 +93,21 @@ commits,) so choose the latest build on the release branch.  (Remember, that
 branch should be frozen.)
 
 Once you find some greens, you can find the git hash for a build by looking at
-the Full Console Output and searching for `githash=`. You should see a line:
+the Full Console Output and searching for `GIT_COMMIT_HASH=`. You should see a line:
```

Comment:
Someone who has cut a release should probably explain how this works.  Is there a call to un-parameterized `set` or somethign that ends up dumping all the env vars?

@ihmccreery